app/views/places/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<main>
+  <div id="place">
+    <picture>
+      <source src="<%= @place.geograph_image_uri %>">
+      <img src="<%= @place.geograph_image_uri %>" alt="<%= @place.title %>" />
```

Comment:
We should be using the `image_uri`s here. The Geograph URIs are just for reference. Sorry should've pointed this out

Reply:
The `image_uri` doesn't have the host/domain etc. Do we have that yet, or are still building bare-bones and I shouldn't worry about it yet?

Reply:
Oh, that's my fault. Lemme send you the env var for your env.development